hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/statistics/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Is this a new package introduced?
Is it really needed.

Reply:
Yes, I created this package for statistics related file. I thought it is good to have this separately instead of keeping it in service package.

Reply:
Makes sense, but then should we have child classes also moved to this new package??

Reply:
Moving all the files to this package results in lot of refactoring. Thats why just added this to new folder.

Reply:
Make sense. Sure, will create a separate work item for this.